src/goto-cc/armcc_cmdline.cpp

```diff
@@ -87,8 +87,8 @@ static const char *options_no_arg[]=
   "--no_implicit_include_searches",
   "--implicit_typename",
   "--no_implicit_typename",
-  "--nonstd_qualifider_deduction",
-  "--no_nonstd_qualifider_deduction",
+  "--nonstd_qualifier_deduction",
+  "--no_nonstd_qualifier_deduction",
```

Comment:
Very nice catch, your change is correct according to http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0472j/chr1359124935242.html